File: xx_docs/selenium/1 - Selenium Testing Tools Cookbook [arquivo-apoio]/Chapter 2/Python/screenshot_test.py
from selenium import webdriver
import time, unittest

class ScreenShotTest (unittest.TestCase):
	def setUp(self)	:
		self.driver = webdriver.Chrome()
		
	def test_screen_shot(self):
		driver = self.driver
		driver.get("http://www.google.com")
		driver.save_screenshot("c:\\tmp\\main_page.png")
		
	def tearDown(self):
		self.driver.close()
		
if __name__ == "__main__":
	unittest.main()

File: XX - Backup Project/xx_docs/selenium/1 - Selenium Testing Tools Cookbook [arquivo-apoio]/Chapter 2/Ruby/screenshot_test.rb
require 'rubygems'
require 'selenium-webdriver'
require 'test/unit'

class ScreenShotTest < Test::Unit::TestCase
	def setup
		
		@driver = Selenium::WebDriver.for :chrome
		@driver.get 'http://www.google.com'
		@verification_errors = []
	end
	
	def test_screen_shot
		@driver.save_screenshot("c:\\tmp\\main_page.png")
	end
	
	def teardown
		@driver.quit
		assert_equal [], @verification_errors
	end
end